cmd/kube-apiserver/app/encryption/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- sig-auth-maintainers
+reviewers:
+- sig-auth
```

Comment:
@liggitt If it's a matter of precedence, other sigs do appear in OWNERS files:
- https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/OWNERS
- https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/disruption/OWNERS

Are you saying you don't want to support sig-auth as an owner?